cluster/gce/util.sh

```diff
@@ -1568,7 +1568,7 @@ EOF
     # Node-only env vars.
     cat >>$file <<EOF
 KUBERNETES_MASTER: $(yaml-quote "false")
-ENABLE_APISERVER_INSECURE_PORT: $(yaml-quote ${ENABLE_APISERVER_INSECURE_PORT:-true})
+ENABLE_APISERVER_INSECURE_PORT: $(yaml-quote ${ENABLE_APISERVER_INSECURE_PORT:-false})
```

Comment:
I hope we tested and ensure this default setting change doesn't carry any other impact.

Reply:
this is what it was before Peng's change.